doc/interop-test-descriptions.md

```diff
@@ -672,14 +679,30 @@ Interaction with flow control is unspecified.
 
 Status: Pending
 
+#### Echo Authenticated Username
+[Echo Authenticated Username]: #echo-username
```

Comment:
The id _must_ be echo-authenticated-username. These ids are generated by GitHub, so we don't get to choose them. You can "View" this file to see the links work.